### PR TITLE
vo_conesearch: Renamed async to vo_async as async is reserved keyword

### PR DESCRIPTION
**Pull request by lupinix**
Renamed async to vo_async as async is reserved keyword starting with Python 3.7 and the import fails as a result: https://docs.python.org/3/whatsnew/3.7.html

Fix for #1178

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -8,7 +8,7 @@
 - MAST: Enable converting list of products into S3 uris [#1126]
 - NASA_ADS: Use new API [#1162]
 - VO_CONESEARCH: Service validator now uses new STScI VAO TAP registry. [#1114]
-
+- VO_CONESEARCH: Renamed async to vo_async as async is reserved keyword starting with Python 3.7. [#1185]
```
Comment (bsipocz) [inline CHANGES]: this doesn't need a changelog as `async` wasn't exposed to the user API.

**Comment by lupinix** [comment]
Shall I remove it?

File: CHANGES (hunk 1 above)
Comment (lupinix) [reply CHANGES]: Shall I remove it?

**Comment by bsipocz** [comment]
Thanks @lupinix! This looks good to me now, but I'll wait for @pllim's review before merging.

**Comment by bsipocz** [comment]
Thanks @lupinix for the contribution!

**Comment by pllim** [comment]
> Would you think they should use it though?

Not sure. Even in cone search, I don't think people use `async` very much.

**Pull request merged**